packages/next/src/export/index.ts

```diff
@@ -238,6 +241,11 @@ export default async function exportApp(
       prerenderManifest = require(join(distDir, PRERENDER_MANIFEST))
     } catch (_) {}
 
+    let appRoutePathManifest: Record<string, string> | undefined = undefined
+    try {
+      appRoutePathManifest = require(join(distDir, APP_PATH_ROUTES_MANIFEST))
```

Comment:
Maybe use `fs.exists` here instead so that if there's an error in the manifest it doesn't accidentally fall through.

Reply:
@timneutkens Should I also make that change to the prerender manifest on line 241 above?

Reply:
Updated in [57cd2b9](https://github.com/vercel/next.js/pull/47022/commits/57cd2b95308a4117471c0bf8adc311150c780d5b)